docs/Getting-Started/Create-bootable-media/bootable-usb-using-linux.md

```diff
@@ -0,0 +1,126 @@
+# Create a Bootable Pop!_OS USB in Linux
+
+Use this guide to create Pop!_OS bootable USB media from Linux:
+
+- Install ISO flashing software.
+- Download the Pop!_OS.iso file.
+- Verify the ISO.
+- Flash the ISO to your USB storage device.
+
+---
+## I. Install Popsicle
+
+![Popsicle Icon](/images/create-bootable-usb-linux/popsicle-icon.png)
+
+Create a bootable USB drive for Pop!_OS using Popsicle. Popsicle is a Linux utility for flashing multiple USB devices in parallel, written in Rust.
+
+Popsicle's installation process may vary depending on your distribution. Skip to the USB creation steps if you're already running Pop!_OS.
+### Ubuntu & Ubuntu-based Distributions
+#### Apt
+
+1. Add the System76 PPA. 
+
+```[bash]
+sudo add-apt-repository ppa:system76/pop
```

Comment:
We do not recommend using this PPA on Ubuntu or any other non-Pop!_OS distributions. On previous releases, it would include all Pop!_OS components (including the kernel, GNOME Control Center, etc) and having the PPA added would cause a user's system to attempt to convert itself to Pop!_OS (probably not entirely successfully) the next time they install regular updates. On 21.10 and above, we don't use the PPA at all, and the PPA does not include Popsicle anyway, so these instructions will not work.

I think recommending Popsicle is perfectly fine, but I would link to https://github.com/pop-os/popsicle/releases/latest and give instructions for using the AppImage since it does not assume any package management capabilities. (Including the Flatpak as an additional option would still be fine.)

Reply:
For future reference, the PPA we support for Ubuntu and Ubuntu-based distributions is `ppa:system76-dev/stable`, although that one does not include Popsicle since it is only meant for hardware support.